src/test/scala/org/apache/spark/sql/delta/DeltaSuite.scala

```diff
@@ -494,6 +494,29 @@ class DeltaSuite extends QueryTest
     }
   }
 
+  test("support removing partitioning") {
+    withTempDir { tempDir =>
+      if (tempDir.exists()) {
+        assert(tempDir.delete())
+      }
+
+      spark.range(100).select('id, 'id % 4 as 'by4)
+        .write
+        .format("delta")
+        .partitionBy("by4")
+        .save(tempDir.toString)
+
```

Comment:
can you also add a check that the partitioning column indeed did exist

Reply:
check was added